gc/base/standard/Scavenger.cpp

```diff
@@ -4444,7 +4444,8 @@ MM_Scavenger::reportGCCycleFinalIncrementEnding(MM_EnvironmentStandard *env)
 		omrtime_hires_clock(),
 		J9HOOK_MM_OMR_GC_CYCLE_END,
 		_extensions->getHeap()->initializeCommonGCData(env, &commonData),
-		env->_cycleState->_type,
+		/* set OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL bit in the cycleType of CycleEnd event in scavenge backout case */
+		(env->getExtensions()->isScavengerBackOutFlagRaised()) ? (cycleType | OMR_GC_CYCLE_TYPE_STATE_UNSUCCESSFUL) : cycleType,
```

Comment:
Hm, this line looks longer than I have imagined it. I'll approve it anyway, but if @dmitripivkine wants to move  the logic back to where it was, i'm fine too.

Reply:
I think this is too heavy and I don't like arithmetic if here. I am not strong on this, but it was better in the separate line

Reply:
ok, let's move it back. sorry for the confusion